recipe/meta.yaml

```diff
@@ -1,21 +1,21 @@
 {% set name = "smesh" %}
-{% set version = "8.3.0.3" %}
+{% set version = "9.6.0_alpha" %}
 {% set sha256 = "bd5bddd509e5120bdd56178c37471984062e8abd36465389d64fb6e1b811a1d4" %}
 
 package:
     name: {{ name }}
     version: {{ version }}
 
 source:
-    fn: {{ name }}_{{ version }}.tar.gz
-    url: https://github.com/LaughlinResearch/SMESH/archive/v{{ version }}.tar.gz
-    sha256: {{ sha256 }}
-    patches:
-        - osx-vtk.patch  # [osx]
-        - linker.patch  # [win]
+    # fn: {{ name }}_{{ version }}.tar.gz
```

Comment:
Some sources say that using `git_url` clones recursively:
- https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#source-from-git
- https://github.com/bioconda/bioconda-recipes/issues/1524

Putting it in the build seems reasonable as well, just thought I'd share in case you would prefer it here.

Reply:
I guess for the package we should not use git but create a release with all the sources included?

Reply:
@looooo i've read that when you create a release with submodules the submodule sources are not included in the resulting zip file.  we could still cut a release but use git and point to the release tag? not 100% how that will work out, but i'm sure we'll figure something out. if needed, in the trelau/SMESH github actions workflow, for a release build i could package up and save the resulting source files and those could probably be downloaded in this feedstock. but, that seems like a last resort.